#Msg for DH parameters of a link
#Type: one of {'revolute', 'prismatic', 'fixed'} depending on method of actuation (fixed for no actuation)
string link_type
float64 d
float64 a
float64 alpha
float64 theta
